torch/_tensor_docs.py

```diff
@@ -1634,9 +1649,9 @@ def callable(a, b) -> number
 An additional dimension of size size is appended in the returned tensor.
 
 Args:
-    dim (int): dimension in which unfolding happens
-    size (int): size of each slice that is unfolded
-    step (int): the step between each slice
+ dim (int): dimension in which unfolding happens
```

Comment:
same as above. you need to replace the tabs with 4 spaces. right now you removed the spaces entirely.